docs/analysis/perfetto-sql-getting-started.md

```diff
@@ -2,8 +2,23 @@
 
 PerfettoSQL is the foundation of trace analysis in Perfetto. It is a dialect of
 SQL that allows you to query the contents of your traces as if they were a
-database. This page introduces the core concepts of PerfettoSQL and provides
-guidance on how to write queries.
+database. This page introduces the core concepts of trace querying with PerfettoSQL
+and provides guidance on how to write queries.
+
+## Overview of Trace Querying
+
+The Perfetto UI is a powerful tool for visual analysis, offering call stacks, timeline views, thread tracks, and slices. However, it also includes a robust SQL query engine (PerfettoSQL) that allows you to extract data programmatically.
```

Comment:
PerfettoSQL is a query language not a query engine. Trace processor is a query engine.